src/PKSim.Assets/PKSimConstants.cs

```diff
@@ -1167,6 +1167,8 @@ public static class MenuNames
          public static readonly string Stop = "Stop";
          public static readonly string BoxWhiskerAnalysis = UI.BoxWhisker;
          public static readonly string TimeProfileAnalysis = UI.TimeProfile;
+         public static readonly string PredictedVsObservedSimulation = "Predicted vs Observed";
```

Comment:
Feels like this is some common string that should be in core?

Reply:
true, it is also duplicated in the MoBi PR, I will move those here to Core

Reply:
done (will be updated with new Core nuget packages)